bin/check-es-version.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#shellcheck disable=SC2086
+
+
+DATABASE_URL=$1
+WAIT=10
+
+function wait_for_reqeust { 
+  for _ in $(seq 1 ${WAIT}); do
+    if ! curl $"DATABASE_URL" > /dev/null 2>&1; then
+      return 0
+    fi
+    sleep 1
+  done
+
+  echo "Unable to reach Elasticsearch server, please check DATABASE_URL and your database server."
+  exit 1
+}
+
+
+# The old, Dockerfile based deploy won't have $TAG set
+if [[ -z "$TAG" ]]; then
+	echo "Your deploy method for docker-kibana is out of date. Please check the latest"
+	echo "instructions at https://github.com/aptible/docker-kibana/blob/master/README.md"
+	exit 1
+else
+	#Check that we're using the right Kibana version for the ES we're connecting to
+
+  KIBANA_VERSION_PARSER="
+  require 'json'
+  es_version = JSON.parse(STDIN.read)['version']['number']
+  print 4.1 if es_version.start_with?('1.')
+  print 4.4 if es_version.start_with?('2.')
+  print 5.0 if es_version.start_with?('5.0.')
+  print 5.1 if es_version.start_with?('5.1.')
+  print 6.0 if es_version.start_with?('6.0.')
+  print 6.1 if es_version.start_with?('6.1.')"
+
+  wait_for_reqeust
+
+  KIBANA_NEEDED_VERSION="$(curl "$DATABASE_URL" 2>/dev/null | ruby -e "$KIBANA_VERSION_PARSER")" 
+
+  if [[ -z "$KIBANA_NEEDED_VERSION" ]]; then
+    echo "Your version of Elasticsearch not supported by this application."
+    exit 1
+  fi
+ 
+  if [[ $TAG != $KIBANA_NEEDED_VERSION ]]; then
+    echo "You're using docker-kibana:${TAG}. To connect to this version of Elasticsearch,"
```

Comment:
The image is not called `docker-kibana`, so this is going to be confusing. Rename this to `aptible/kibana:${TAG}`.

Besides, I would suggest rewording this to indicate that the reason they need to update to connect is because Elasticsearch is not compatible with that version.